samples/matter/common/src/bridge/bridged_device_data_provider.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "bridged_device_data_provider.h"
+
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_DECLARE(app, CONFIG_CHIP_APP_LOG_LEVEL);
+
+void BridgedDeviceDataProvider::NotifyReachableStatusChange(bool isReachable)
```

Comment:
Nit: It could return CHIP_ERROR to handle CHIP_ERROR_NO_MEMORY case if `New` failed.